src/cmd/src/lib.rs

```diff
@@ -114,6 +122,25 @@ pub fn log_versions(version: &str, short_version: &str, app: &str) {
     log_env_flags();
 }
 
+#[cfg(target_os = "linux")]
+pub fn create_resource_limit_metrics(app: &str) {
+    if let Some(cpu_limit) = get_cpu_limit() {
+        info!(
+            "GreptimeDB start with cpu limit in millicores: {}",
+            cpu_limit
+        );
+        CPU_LIMIT.with_label_values(&[app]).set(cpu_limit);
+    }
+
+    if let Some(memory_limit) = get_memory_limit() {
+        info!(
+            "GreptimeDB start with memory limit in bytes: {}",
+            memory_limit
+        );
+        MEMORY_LIMIT.with_label_values(&[app]).set(memory_limit);
+    }
+}
```

Comment:
We can provide an empty method for non-linux os. Then it should be possible to use `create_resource_limit_metrics()` without cfg.

What's more, the `stat` crate can return None for non-linux os, so we can use this crate without cfg.